Etabs_Adapter/ETABSAdapter.cs

```diff
@@ -84,7 +84,9 @@ public ETABSAdapter(string filePath = "", EtabsSettings etabsSetting = null, boo
             AdapterIdFragmentType = typeof(ETABSId);
             BH.Adapter.Modules.Structure.ModuleLoader.LoadModules(this);
             SetupDependencies();
+            SetupPriorities();
             SetupComparers();
+            m_AdapterSettings.HandlePriorities = true;
```

Comment:
Error when building (after having build [#393](https://github.com/BHoM/BHoM_Adapter/pull/393)):
Could not find out myself why, @GCRA101 
![image](https://github.com/user-attachments/assets/604de418-37b7-4721-9c41-b6c001c6fb2b)

Reply:
@HugoVanLooveren , all sorted now. Code updated in both BHoM_Adapter and ETABS_Toolkit accordingly. ;)